src/doc/en/installation/source.rst

```diff
@@ -446,20 +446,34 @@ If you don't want conda to be used by sage, deactivate conda (for the current sh
   operating system, or its own compilers.
 
 
-Tcl/Tk
-^^^^^^
+Tcl/Tk (and system's Python)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to use `Tcl/Tk <https://www.tcl.tk/>`_ libraries in Sage, and you
+are going to use your OS's Python3 as Sage's Python, you merely need to install
+its **Tkinter** module.  On Linux systems, it is usually provided by the
+**python3-tk** or a similarly named (e.g. **python3-tkinter**) package,
+which can be installed using::
+
+    $ sudo apt-get install python3-tk
```

Comment:
-1 on adding ad-hoc installation commands for 1 platform. We can do better - dummy package with system package info.

Reply:
I'll take care of this in a follow up.